src/components/Tile/Tile.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import { DataAttributes } from "../../helpers/Attributes";
+
+export interface TileProps extends React.AriaAttributes, DataAttributes {
+  /**
+   * A href link address that is passed into the Tile component
+   */
+  link: string;
+  /**
+   * A title wrapped in an anchor tag.
+   */
+  title: string | null;
+  /**
+   * An array of type TileDetailList elements. Each element consisting of a key/value pair.
+   */
+  details: TileDetailList[];
+  /**
+   * Value for the mailto input.
+   */
+  email?: string | null;
+}
+
+export interface TileDetailList {
+  /**
+   * The key used to identify an individual property in the array
+   */
+  key: string;
+
+  /**
+   * The value associated with a property.
+   */
+  value: string;
+}
+
+export const Tile = (props: TileProps): React.ReactElement => {
+  let mailto = null;
```

Comment:
Again, null is slightly different in js. You should be setting things to undefined over null.
In this case you can just write `let mailto;` and mailto will be initialised as undefined.